models/public/netvlad/accuracy-check.yml

```diff
@@ -0,0 +1,16 @@
+models:
+  - name: netvlad
+    launchers:
+      - framework: dlsdk
+        adapter: reid
+    datasets:
+      - name: pitts30k_val
+
+        preprocessing:
+          - type: rgb_to_bgr
```

Comment:
looks confusing, could you please move reader to this config instead of definitions?

Reply:
Done